Sources/SwiftDoc/Symbol.swift

```diff
@@ -43,7 +43,11 @@ public final class Symbol {
 
         if let `extension` = `extension`,
             `extension`.modifiers.contains(where: { $0.name == "public" }) {
-            return true
+
+            let isNotLimitedAccessModifier = { (modifier: Modifier) in
+                modifier.name != "internal" && modifier.name != "fileprivate" && modifier.name != "private"
+            }
+            return api.modifiers.allSatisfy(isNotLimitedAccessModifier)
```

Comment:
**nitpick:** as this closure is not reused anywhere else, why not inline it? The explicit argument type signature wouldn't be needed then:

```suggestion
            return api.modifiers.allSatisfy { modifier in
                modifier.name != "internal" && modifier.name != "fileprivate" && modifier.name != "private"
            }
```

Reply:
Sure thing. Originally I wanted to extract this out of this place but then figured out that it’s not necessary and it stayed like this.

Reply:
Thank you!